### ISSUE TITLE
Training dataset of rxnmapper?

### ISSUE THREAD

**Issue by Sunggi99** [rxn4chemistry/rxnmapper#23]
I wonder whether the trained rxnmapper model was trained by the 1k data if USPTO50K described in the paper.
 If not, is it possible to provide the model trained by the 1k reactions? 
 Any help about above questions would be greatly appreciated.

**Comment by pschwllr** [comment]
The 1k data was just used to evaluate the different trained models and select one to test on the remaining USPTO50k reactions. They were not part of the training.

The model was trained using an unsupervised task (masked language modelling) on the USPTO reactions without atom-mapping that you find here: https://ibm.ent.box.com/v/RXNMapperData/folder/112973143084.
